ubcpi/static/css/ubcpi.css

```diff
@@ -349,17 +479,66 @@ textarea.pi-options {
 }
 
 .correct-answer-option {
-    margin-bottom: 20px;
+    margin-top: 20px;
+    margin-bottom: 5px;
+}
+
+.show-correct {
```

Comment:
It may be worth just prefacing your selectors with "ubcpi-" to guarantee
uniqueness.

On Tue, Dec 15, 2015 at 1:36 PM, Richard Tape <notifications@github.com>
wrote:

> In ubcpi/static/css/ubcpi.css
> <https://github.com/ubc/ubcpi/pull/55#discussion_r47677333>:
>
> > @@ -349,17 +479,66 @@ textarea.pi-options {
> >  }
> >
> >  .correct-answer-option {
> > -    margin-bottom: 20px;
> > +    margin-top: 20px;
> > +    margin-bottom: 5px;
> > +}
> > +
> > +.show-correct {
>
> Darn it, good catch. I've just made a PR to our own repo to fix up these
> potential clashes. We'll get it into this merged PR asap. Thanks for
> pointing it out.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ubc/ubcpi/pull/55/files#r47677333>.
>